<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HorseWidget</class>
 <widget class="QWidget" name="HorseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>350</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>开启强化模式,仅限本次扫描</string>
    </property>
   </widget>
   <widget class="QLabel" name="subtitle">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>顶起木马查杀, 时刻保持电脑健康</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbXinrenqu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>快速查杀</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbAnweizhi">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbHuifuqu">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShouji">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>390</y>
      <width>60</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="FocusWidget" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="WaveBallWidget" name="wallBallWidget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="tbWinPE">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>390</y>
      <width>60</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>上次扫描共10项威胁未处理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="tbQuanpan">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbXitong">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>60</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbShangbanqu">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>40</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveBallWidget</class>
   <extends>QWidget</extends>
   <header location="global">customviews/waveballwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FocusWidget</class>
   <extends>QPushButton</extends>
   <header location="global">customviews/focuswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
